forc-util/src/lib.rs

```diff
@@ -207,6 +207,16 @@ pub fn println_green(txt: &str) {
     println_std_out(txt, TermColor::Green);
 }
 
+pub fn println_light_blue(txt: &str) {
```

Comment:
Should we name this as `print_light_blue` rather than `println_light_blue` as it might be confusing later on?